Meetings/2022-02-16=meeting-minutes.md

```diff
@@ -0,0 +1,72 @@
+# Maintainers' Meeting 2022-02-16
+
+## Meeting Attendees
+
+- Rob Prouse - @rprouse
+- Mattias Karlsson - @devlead
+- Andrew White - @AndrewTriesToCode
+- Dylan Perks - @Perksey
+- Glenn Watson - @glennawatson
+- James Newton-King - @JamesNK
+- James Turner - @Turnerj
+- Michael Hawker
+- Pascal Berger - @pascalberger
+- Robin Krom
+- Tanner Gooding - @tannergooding
```

Comment:
Tom Pappas was in the meeting according to the image capture I made

Reply:
Tom is listed below in the Observers section. He works for the foundation.